operator/params.yaml

```diff
@@ -701,3 +701,23 @@ JVM_OPT_MAX_GC_PAUSE_MILLIS:
 JVM_OPT_G1R_SET_UPDATING_PAUSE_TIME_PERCENT:
   description: "Have the JVM do less remembered set work during STW, instead preferring concurrent GC. Reduces p99.9 latency."
   default:
+
+READINESS_CHECK_INITIAL_DELAY:
+  description: "An amount of time in seconds to wait before starting the readiness check attempts."
+  default: "0"
+
+READINESS_CHECK_PERIOD:
+  description: "The period in seconds to wait after the last readiness check has completed to start the next check."
+  default: "5"
+
+READINESS_CHECK_TIMEOUT:
+  description: "An amount of time in seconds to wait for a readiness check to succeed."
+  default: "60"
+
+READINESS_SUCCESS_THRESHOLD:
```

Comment:
The other params are prefixed with `READINESS_CHECK`. Can you do the same for the others or drop `CHECK` from the first two params?